project/src/main/webapp/candidate.html

```diff
@@ -19,6 +19,65 @@ <h2>Voter Central Candidate Page</h2>
       <div class="candidate-page-info" id="official-info-container">
       </div>
       <div class="candidate-page-info" id="news-articles-container">
+        <article name="news-article-1">
+          <h3>News article title</h3>
+          <a href="URL">Source</a>
+          <h5>Publisher</h5>
+          <time>Published date</time>
+          <div name="news-article-content">
+            <p>Content [first 100 words]</p>
+            <a href="URL">Read the full article</a>
+          </div>
+          <hr>
+        </article>
+        <article name="news-article-2">
+          <h3>Alexandria Ocasio-Cortez faces primary challenge from Michelle
+              Caruso-Cabrera</h3>
+          <a href="https://www.cnn.com/2020/06/23/politics/aoc-ny-primary-14th-district/index.html">
+              Source</a>
+          <h5>CNN</h5>
+          <time>11:21 AM June 23, 2020 EST</time>
+          <div name="news-article-content">
```

Comment:
It seems like you tend to prefer the "name" attribute while I tend to use the "id" attribute more. I don't necessarily have a preference, but should we worry about keeping it consistent between us?

Reply:
Oh I think I meant to use "class" here. Sorry about that! The attribute here was more for readability purposes, than for referencing the element. We can also style all news article content by selecting the class. I imagined constructing the entire news article section in JavaScript, so I wouldn't use getElementById() to find the element and then fill in content.
I agree that we should be consistent. I normally use "id" as well

Reply:
And for the article elements, the attribute should be "id" instead of "name". I just noticed that. I'll make the change!